openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -1799,7 +1799,7 @@ The following additional Credential Issuer metadata are defined for this Credent
   * `@context`: REQUIRED. Array as defined in [@VC_DATA], Section 4.1.
   * `type`: REQUIRED. Array designating the types a certain credential type supports according to [@VC_DATA], Section 4.3.
   * `credentialSubject`: OPTIONAL. An object containing a list of name/value pairs, where each name identifies a claim offered in the Credential. The value can be another such object (nested data structures), or an array of such objects. To express the specifics about the claim, the most deeply nested value MAY be an object that includes a following non-exhaustive list of parameters defined by this specification:
-      * `mandatory`: OPTIONAL. Boolean which when set to `true` indicates the claim MUST be present in the issued Credential. If the `mandatory` property is omitted its default should be assumed to be `false`.
+      * `mandatory`: OPTIONAL. Boolean which, when set to `true`, indicates that the Credential Issuer will always include this claim in the issued Credential. If set to `false`, the Credential Issuer can choose not to include the claim. If the `mandatory` parameter is omitted, the default should be assumed to be `false`.
```

Comment:
"the Credential Issuer can choose not to include the claim" potentially seems to suggesting the decision is only made by the issuer; I was presuming the decision could also be made by the user or based on what the wallet requests?

Perhaps something more like "the Credential Issuer may issue Credentials that do not include the claim" ?

```suggestion
      * `mandatory`: OPTIONAL. Boolean which, when set to `true`, indicates that the Credential Issuer will always include this claim in the issued Credential. If set to `false`, the Credential Issuer may issue Credentials that do not include the claim. If the `mandatory` parameter is omitted, the default should be assumed to be `false`.
```

Reply:
This might be more accurate:

```suggestion
 * `mandatory`: OPTIONAL. Boolean which, when set to `true`, indicates that the Credential Issuer will always include this claim in the issued Credential. If set to `false`, the claim is not included in the issued Credential if the wallet did not request the inclusion of the claim in the Authorization Request, or if the Issuer chose to not include the claim. If the `mandatory` parameter is omitted, the default should be assumed to be `false`.
```